desktop/ui/src/main/java/org/datacleaner/windows/AnalysisJobBuilderWindowImpl.java

```diff
@@ -505,10 +512,10 @@ public void updateStatusLabel() {
                     setStatusLabelWarning();
                 }
             } catch (Exception ex) {
+                executeable = false;
```

Comment:
Ah, too early in the morning. Didn't notice it was also being used for other purposes. But it still partially holds: If all branches are supposed to set the executable flag to false, why do in inside the if construct at all?